_rules/first-focusable-elements-are-links-to-content-e53727.md

```diff
@@ -0,0 +1,1000 @@
+		<title>The Three Kingdoms, Chapter 1</title>
+	</head>
+	<body>
+		<nav>
+			<a href="#bio-translator">Skip to translator's biography</a>
+			<a href="#about-book">Skip to information about the book</a>
+			<a href="https://www.w3.org/">Check out the W3C</a>
+			<a href="#main">Skip to text</a>
+		</nav>
+
+		<aside id="bio-translator">
+			<h1>About the translator</h1>
+			<p>Yu Sumei is a professor of English at East China Normal University.</p>
+		</aside>
+		<aside id="about-book">
+			<h1>About the book</h1>
+			<p>The Romance of the Three Kingdoms is a 14th century historical novel.</p>
+		</aside>
+
+		<main id="main">
+			<h1>Three Heroes Swear Brotherhood at a Feast in the Peach Garden</h1>
+			<p>
+				Unity succeeds division and division follows unity. One is bound to be replaced by the other after a long span
+				of time.
+			</p>
+			<a href="/test-assets/bypass-blocks-cf77f2/chapter1.html">Read Chapter 2</a>
+		</main>
+	</body>
+</html>
+```
+
+#### Failed Example 4
+
+In this document, the first three [focusable][] elements in this [document][] form an [initial segment][] that fulfills most conditions but none of these elements are [included in the accessibility tree][].
+
+```html
+<html lang="en">
+	<head>
+		<title>The Three Kingdoms, Chapter 1</title>
+	</head>
+	<body>
+		<nav>
+			<a href="#bio-translator" aria-hidden="true">Skip to translator's biography</a>
+			<a href="#about-book" aria-hidden="true">Skip to information about the book</a>
+			<a href="#main" aria-hidden="true">Skip to text</a>
+		</nav>
+
+		<aside id="bio-translator">
+			<h1>About the translator</h1>
+			<p>Yu Sumei is a professor of English at East China Normal University.</p>
+		</aside>
+		<aside id="about-book">
+			<h1>About the book</h1>
+			<p>The Romance of the Three Kingdoms is a 14th century historical novel.</p>
+		</aside>
+
+		<main id="main">
+			<h1>Three Heroes Swear Brotherhood at a Feast in the Peach Garden</h1>
+			<p>
+				Unity succeeds division and division follows unity. One is bound to be replaced by the other after a long span
+				of time.
+			</p>
+			<a href="/test-assets/bypass-blocks-cf77f2/chapter1.html">Read Chapter 2</a>
+		</main>
+	</body>
+</html>
+```
+
+#### Failed Example 5
+
+In this document, the first [focusable][] element form an [initial segment][] that fulfills Expectation 1 but not Expectation 2 because there is no link to the [main block of content][]. The first three [focusable][] elements form an [initial segment][] that fulfills Expectation 2 but not Expectation 1 because the second [focusable][] element is not [included in the accessibility tree][]. Thus, no [initial segment][] fulfills both Expectations.
+
+```html
+<html lang="en">
+	<head>
+		<title>The Three Kingdoms, Chapter 1</title>
+	</head>
+	<body>
+		<nav>
+			<a href="#bio-translator">Skip to translator's biography</a>
+			<a href="#about-book" aria-hidden="true">Skip to information about the book</a>
+			<a href="#main">Skip to text</a>
```

Comment:
Do you mean the initial segment only contains one link? according to the definition of the initial segment we have a well set of elements let me say `<nav>`, as the second element became hidden because of `<aria-hidden>` then we have only two links in the initial set and one of them is main. 
Correct me if I am interpreting is wrong.

Reply:
There are, of course, several initial segment and we need to find one which passes both expectations.

* The initial segment with only one element does not pass Expectation 2, whatever the chosen semantic segmentation (no link to main).
* The initial segment with 2 elements does not pass Expectation 1 because it contains an `aria-hidden` element.
* The initial segment with 3 (or more) elements does not pass Expectation 1 (same as with only 2 elements, it contains an `aria-hidden` element).

Thus, no initial segment passes both Expectations and the rule fails.

The initial segment, as built by Expectation 1, is an initial segment **of focusable elements**, compare:
1. "There exists an initial segment of focusable elements, such that they are all included in the accessibility tree" (current formulation)
2. "There exists an initial segment of the elements that are (focusable and included in the accessibility tree)" (parenthesis added for clarity)

In the first case, we consider all focusable elements (independently of inclusion in the accessibility tree) and thus cannot "jump" over the `aria-hidden` element when building the initial segment (and the presence of this element makes any segment of length 2 or more fail Expectation 1 because it fails the immediate check).
In the second case, we start by filtering the non-accessible links.

---
Now, I do agree that this example is likely satisfying 2.4.1… I'm starting to doubt the structure of these rules for G1 ad G124 anyway… More on that in another message, I need to group my remarks together 😕